docs/component/oa-verify.md

```diff
@@ -4,8 +4,188 @@ title: Verification SDK (javascript)
 sidebar_label: Verification SDK (javascript)
 ---
 
-Overview
+The OpenAttestation Verification SDK (javascript) is a npm module that allows you to  verify [wrapped document](/docs/component/open-attestation) programmatically. This is useful if you are building your own API or web components. Some common use cases where you need this module:
+- [Verifying a document](#verifying-a-document)
+- [Building custom verifiers](#custom-verification)
+- [Building custom validation](#custom-validation)
 
-## Header
+This module does not provide the following functionality:
+- Programmatic wrapping of OA documents (refer to [OpenAttestation Wrapper SDK (javascript)](/docs/component/open-attestation))
+- Encryption or decryption of OA documents (refer to [Encryption SDK (javascript)](/docs/component/oa-encryption))
+- Programmatic issuance/revocation of document on the Ethereum blockchain
 
-TBD
+## Installation
+
+1. Install [Node.js](https://nodejs.org/en/)
+1. Install the library: `npm install @govtechsg/oa-verify`
+
+## Verifying a document
+
+A verification happens on a wrapped document, and it consists of answering to 4 questions:
+- Has the document been tampered with ?
+- Has the document been issued on the Blockchain ?
```

Comment:
Will suggest to merge issuance and revocation together:

"Has is the issuance state of the document valid?"

This will make it consistent with the verifier design (https://github.com/Open-Attestation/adr/blob/master/verifier.md#architecture)

Reply:
done